Vates/VatesAPI/src/vtkSplatterPlotFactory.cpp

```diff
@@ -252,7 +254,8 @@ void vtkSplatterPlotFactory::doCreate(
 
   // The list of IDs of points used, one ID per point, since points
   // are not reused to form polygon facets, etc.
-  vtkIdType *ids = new vtkIdType[numPoints];
+    std::vector<vtkIdType> ids; 
+    ids.assign(numPoints, 0);
 
   // Only one scalar for each cell, NOT one per point
   vtkFloatArray *signal = vtkFloatArray::New();
```

Comment:
Raw owning pointer (and now delete?). Changed correctly in other functions below.